css/style.css

```diff
@@ -265,6 +265,7 @@ body {
 .img-btn {
   max-width: 100%;
 }
+
```

Comment:
- [ ] As this is a mobile-first design, which means designing for mobile before designing for desktop or any other device, I think it would be better to change from ```@media (min-width: 768px)``` to ```@media (max-width: 768px)``` and this  will apply the styles within it only when the maximum width of the viewport is 768 pixels or less 😎😎

Reply:
@TuyishimireEric
Thank you for you detail comments and yes I have tried both the @media (min-width: 768px) to @media (max-width: 768px) because I started mobile first with minimum width 375 even though it doesn't mentioned when I add the max-width of the Mobile-first respond will stop and just started for max-width screen device. I want to test all the effects, It is just working. I appreciate and respect your comment Thank you very much.